privatePackages/assert-project/src/index.ts

```diff
@@ -39,13 +39,13 @@ export default (t: Test, projectPath: string, encodedRegistryName?: string) => {
       }
       return path.join(await project.getStorePath(), pkgFolder, 'package')
     },
-    async storeHas (pkgName: string, version?: string) {
-      const pathToCheck = await project.resolve(pkgName, version)
+    async storeHas (pkgName: string, version?: string, store?: string) {
+      const pathToCheck = store || await project.resolve(pkgName, version)
```

Comment:
as far as I understand, with this change, the function will only check if the store exists, not if the package/version exists in the store

Reply:
Oops, that is correct. My bad